fiorella/src/sections/product.liquid

```diff
@@ -131,3 +124,11 @@
     ]
   }
 {% endschema %}
+
+<style type="text/css">
+  #Quantity{
```

Comment:
There's a missing space before the curly brace and try not to use ids to target elements in CSS.